includes/admin/class-pureclarity-settings-page.php

```diff
@@ -147,6 +147,15 @@ private function get_general_fields() {
 			'value_cb'    => 'is_nightly_feed_enabled',
 		);
 
+		$feeds_logging_enabled_checkbox = array(
+			'name'        => 'pureclarity_feed_debug_logging',
+			'label_for'   => 'pureclarity_feed_debug_logging',
+			'label'       => 'Enable Debug Logging',
+			'type'        => 'checkbox',
+			'description' => 'Check to enable extra logging for feeds & deltas, to show what items are being process / skipped. Please only enable as a temporary measure, as it can produce large log files.',
```

Comment:
process**ed**

Reply:
please only enable when requested by PureClarity support ?

Reply:
how about "Please only enable if you are troubleshooting the feeds/deltas", and I'll ass some info tot he troubleshooting docs to recommend using it for certain things? (means we can promote independence :) )